agent/consul/config_replication.go

```diff
@@ -228,9 +232,14 @@ func (s *Server) replicateConfig(ctx context.Context, lastRemoteIndex uint64, lo
 			return 0, true, nil
 		}
 		if err != nil {
-			return 0, false, fmt.Errorf("failed to update local config entries: %v", err)
+			merr = multierror.Append(merr, err)
```

Comment:
do we want to preserve the error message here?

Reply:
same